runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -1215,6 +1223,32 @@ TR::CompilationInfoPerThreadRemote::getCachedResolvedMethod(TR_ResolvedMethodKey
    return false;
    }
 
+/**
+ * @brief Method executed by JITServer to retrieve methodinfo from the resolved method cache
+ *
+ * @param key Identifier used to identify a resolved method in resolved methods cache
+ * @param unresolvedInCP The unresolvedInCP boolean value of interest, set by this API
+ * @return returns the method info
+ */
+TR_ResolvedJ9JITServerMethodInfo
+TR::CompilationInfoPerThreadRemote::getCachedMethodInfo(TR_ResolvedMethodKey key, bool *unresolvedInCP)
+   {
+   TR_ResolvedMethodCacheEntry methodCacheEntry = {0};
+
+   getCachedValueFromPerCompilationMap(_resolvedMethodInfoMap, key, methodCacheEntry);
+
+   uint32_t vTableSlot = methodCacheEntry.vTableSlot;
+   auto methodInfoStruct = methodCacheEntry.methodInfoStruct;
+   TR_ResolvedJ9JITServerMethodInfo methodInfo = make_tuple(methodInfoStruct,
+      methodCacheEntry.persistentBodyInfo ? std::string((const char*)methodCacheEntry.persistentBodyInfo, sizeof(TR_PersistentJittedBodyInfo)) : std::string(),
+      methodCacheEntry.persistentMethodInfo ? std::string((const char*)methodCacheEntry.persistentMethodInfo, sizeof(TR_PersistentMethodInfo)) : std::string(),
+      methodCacheEntry.IPMethodInfo ? std::string((const char*)methodCacheEntry.IPMethodInfo, sizeof(TR_ContiguousIPMethodHashTableEntry)) : std::string());
+
+   if (unresolvedInCP)
+      *unresolvedInCP = (methodInfoStruct.remoteMirror == NULL);
```

Comment:
This code is outdated, since we concluded that this is not how `unresolvedInCP` should be computed.
Instead, `unresolvedInCP` in this method should be replaced with `TR_YesNoMaybe unresolvedInCache` and just set it to the cached value.